WebDriverAgentLib/Categories/XCElementSnapshot+FBHelpers.m

```diff
@@ -97,11 +99,14 @@ - (XCElementSnapshot *)fb_parentMatchingType:(XCUIElementType)type
   return snapshot;
 }
 
-- (XCElementSnapshot *)fb_parentMatchingOneOfTypes:(NSArray<NSNumber *> *)types
+- (XCElementSnapshot *)fb_findVisibleParentMatchingOneOfTypesAndHasMoreThanOneChild:(NSArray<NSNumber *> *)types
```

Comment:
I remember I thought about it passing extra argument that will make distinction but I was not sure how that useful be. But yes we can do for later refactoring :). If you really want it , I can do it but what would be method signature?